### DISCUSSION TITLE
account is suspended without any notification

### DISCUSSION THREAD

**Discussion by MaulidFIshlah** [community/community#135535] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Has anyone ever experienced a github account being suspended? but there was no notification from github, why did it get suspended, my github account suddenly disappeared and was suspended, when I checked my email there was no notification from github, it just got suspended, and there was no notification of the error that was made until it was suspended, and I don't feel like I did anything that violated the rules, can anyone help with this?

this is my lost github account https://github.com/MaulidFajrulIshlah

**Comment by devshDotTech** [comment]
My account got suspended as well due to violation of "community guidelines" last week, and i got no mail, as of what guideline i break. I posted a ticket about account recovery, but i haven't got any reply from them, all my work was on my github account, all the code of deployed web apps was on github and i have lost all of it, try to file a complaint ticket, and lets hope for the best

**Comment by ebndev** [comment]
Hi @MaulidFIshlah,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
